docs/en/remote-flow.rst

```diff
@@ -444,45 +444,62 @@ Where the following parameters are used:
     - Unique identifier provided by the Relying Party within the Authorization Request.
 
 
-Below is a non-normative example of the ``vp_token`` decoded content, represented in the form of JWS header and payload, separated by a period:
+The items contained in the ``vp_token`` array are Verifiable Presentations of Credentials.
+Both SD-JWT and mdoc CBOR provide indications for the presentation, according to their specifications.
 
-.. code-block:: text
+SD-JWT Presentation
+-------------------
 
-   {
-     "alg": "ES256",
-     "typ": "JWT"
-   }
-   .
-   {
-     "iss": "vbeXJksM45xphtANnCiG6mCyuU4jfGNzopGuKvogg9c",
-     "jti": "3978344f-8596-4c3a-a978-8fcaba3903c5",
-     "aud": "https://relying-party.example.org/response_uri",
-     "iat": 1541493724,
-     "exp": 1573029723,
-     "nonce": "2c128e4d-fc91-4cd3-86b8-18bdea0988cb",
-     "vp": "<Issuer-Signed-JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>"
-   }
+SD-JWT defines how an Holder can present a Credential to a Verifier proving the legitimate possession
+of the Credential. For doing this the Holder MUST include the ``KB-JWT`` in the SD-JWT,
+by appending the ``KB-JWT`` at the end of the of the SD-JWT, as represented in the example below:
```

Comment:
As commented above the `KB-JWT` is in the version 5 or above of sd-jwt spec. So we have refers three different versions of sd-jwt spec here. My concern is reporting different versions of the specification. Is it a safe approach?

Reply:
SD-JWT aligned with 07

I suggest to provide suggestion to speed up the revision phase. The problem statement alone is unfortunately  not enough